src/mail/mjml/invitation.mjml

```diff
@@ -0,0 +1,57 @@
+<mjml>
+  <mj-include path="./partial/header.mjml" />
+
+  <mj-body mj-class="bg--blue-100">
+    <mj-wrapper css-class="wrapper" padding="0 40px 40px 40px">
+      <mj-section background-url="{% base64_static 'images/mail-header-background.png' %}" background-size="cover" background-repeat="no-repeat" background-position="0 -30px">
+        <mj-column>
+            <mj-image align="center" src="{% base64_static 'images/logo-suite-numerique.png' %}" width="250px" align="left" alt="{%trans 'La Suite Numérique' %}" />
+        </mj-column>
+      </mj-section>
+      <mj-section mj-class="bg--white-100" padding="30px 50px 60px 50px">
+        <mj-column>
+          <mj-text font-size="14px">
+            <p>{% trans "Invitation to join a team" %}</p>
+          </mj-text>
+
+          <!-- Welcome Message -->
+          <mj-text>
+            <h1>{% trans "Welcome to" %} <strong>Desk</strong></h1>
```

Comment:
I think the app should have the same name for all the languages, because it will be complicated with the images.
The title is `Welcome to Desk` when the image says `Equipes`.
![image](https://github.com/numerique-gouv/people/assets/25994652/4510ba6a-f4be-48eb-b1bb-c25ebc16d7ba)

Reply:
I followed the design! But true

Reply:
The more important part is to ship smth, we'll iterate on the content itself later on.

Reply:
Ok, after we shouldn't follow the design blindly. It is still kind of ok because we are in dev but soon we will not be, so either the developers (and the reviewers) have some liberty to fix by themself the inconsistency, or we must have a process of design review, before it is shipped to the developer.

Reply:
I didn't follow the design blindly. For example, I added a link pointing to desk to invite people, which was missing and necessary to get the flow working.

Your comment is totally relevant, but the feature is functional. I am pretty we gonna iterate on this copywriting quite a few time before going to production. Our goal, and what seems to be the next week target, it to ship smth in staging, we are far from production.

Reply:
Some feature in the frontend are totally not ready for production, and that doesn't block us from merging them 🤗 why it would be different there?

Reply:
[Issue opened](https://github.com/numerique-gouv/people/issues/142) + comment on the Figma